.github/workflows/docker-build-tag.yml

```diff
@@ -24,13 +24,13 @@ jobs:
           strip_v: true
 
       - name: Check out GitHub repo
-        uses: actions/checkout@v2
```

Comment:
Should we update actions in other workflows too?

Reply:
added